app/assets/javascripts/admin/components/ace-editor.js.es6

```diff
@@ -6,6 +6,8 @@ export default Ember.Component.extend({
   classNames: ['ace-wrapper'],
   _editor: null,
   _skipContentChangeEvent: null,
+  _unusedStashSelf: null,
+  stashSelf: Em.computed.alias('_unusedStashSelf'),
```

Comment:
I was using it to have a way to call `aceComponent.trigger('resize')` from the view:admin-plugins-explorer. Is there a better way to do that?